test/geocoders/nominatim.py

```diff
@@ -80,15 +80,26 @@ def test_geocode_language_parameter(self):
         """
         Nominatim.geocode using `language`
         """
+        input = "Mohrenstrasse Berlin"
```

Comment:
not requesting that you change it, but in the future let's use "generic" test data in public repos like this one (so pick the reichstag's address, or berghain's address, or something, let's keep dh as far out of it as possible)